nixos/modules/security/sudo.nix

```diff
@@ -47,6 +63,80 @@ in
         '';
     };
 
+    security.sudo.extraRules = mkOption {
+      description = ''
+        Define specifc rules to be in the <filename>sudoers</filename> file.
+      '';
+      default = [];
```

Comment:
Can you add also an example here?

Reply:
Also has a minor typo (`specifc`)

Reply:
@adisbladis Thanks, fixed it very sneakily in the new commit. :)

Reply:
@adisbladis Psst, fixed it in a new commit. :)